tests/e2e/local/minikube/install_prereqs_centos.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Check if yum is installed
+if ! yum --help > /dev/null; then
+    echo "yum not installed. Please install it manually and run this script again."
+    exit 1
+fi
+sudo yum --quite -y update
+
+# Check if rpm is installed
+if ! rpm --help > /dev/null; then
+    echo "rpm not installed. Please install it and run this script again."
+    exit 1
+fi
+
+#Install Curl
+echo "Checking and Installing Curl as required"
+if ! curl --help > /dev/null; then
+    sudo yum -y install curl
+    if ! curl --help > /dev/null; then
+      echo "curl could not be installed. Please install it manually and run this script again."
+      exit 1
+    fi
+fi
+
+#Install Kvm2
+echo "Installing KVM2 as required"
+sudo modprobe kvm > /dev/null 2>&1
+sudo yum install -y qemu-kvm qemu-img virt-manager libvirt libvirt-python libvirt-client libguestfs-tools virt-install virt-viewer bridge-utils
```

Comment:
the downside of this is libvirt is enabled by default (by libvirt sadly).  Files are not simply installed, but instead, the startup behavior of the system is modified.   Can you add a generalized check at the top of this file that explains what changes will be made to the system during the execution and permit them to opt out of running the script.

My main concern is developer environments, not CI/CD (which it appears this PR is targeted at).

Reply:
So adding one line at the top of this file telling libvirt will be enabled when system boots?

Reply:
I just added one line at the top of this file:

```
# NOTE: libvirtd will be enabled when system boots.
```

PTAL.

Reply:
I wonder if we can do a check first and install only when it is required.

@JimmyCYJ , was that considered when your wrote install_prereqs_debian.sh?

Reply:
@hklai 

Do you mean check if `qemu-kvm qemu-img virt-manager libvirt libvirt-python libvirt-client libguestfs-tools virt-install virt-viewer bridge-utils` are installed first?

Reply:
For example, 

```
yum list installed {package_name}
```

OR

```
dpkg -l {package_name}
```

??

Reply:
@hklai 

I sent out a PR on "do a check first and install only when it is required.", xref: #8521. Please to see if you are happy with that. If #8521 got merged, then I can rebase this PR.

Reply:
Thanks and commented in that PR.

Reply:
Rebased to the HEAD as #8521 went in.